include/seastar/net/socket_defs.hh

```diff
@@ -123,6 +133,14 @@ struct ipv6_addr {
     }
 };
 
+struct ud_addr {
```

Comment:
s/ud_addr/unix_domain_addr/

IMHO, a type as a part of interface deserves a better name.

Reply:
fixed